src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574661,15 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040000
+name: Sharpin-related autoinflammatory syndrome
+def: "An autoinflammatory disease in which the cause of the disease is a variation in the Sharpin gene." [https://orcid.org/0000-0002-0146-1162, MONDO:patterns/disease_series_by_gene, PMID:38609546]
+synonym: "sharpenia" EXACT [https://orcid.org/0000-0002-0146-1162, PMID:38609546]
+is_a: MONDO:0019751 {source="PMID:38609546", source="https://orcid.org/0000-0002-0146-1162"} ! autoinflammatory syndrome
+property_value: http://purl.org/dc/elements/1.1/date "2024-07-02T15:47:35Z" xsd:dateTime
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+
```

Comment:
- please add the term tracker item: https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#term-tracker-item
- you can add a logical definition here, I think you know how to do that because you did it on another PR. Is the Sharpin gene in Mondo or do you need to import it? Here are the instructions: https://mondo.readthedocs.io/en/latest/editors-guide/import-terms-for-logical-axioms/
I can help if needed

Reply:
Added the term tracker - will return later to try the logical definition part.

Reply:
let me know if you need help, I am happy to help :)